tests/src/e2e/eth_bridge_tests.rs

```diff
@@ -137,11 +137,8 @@ async fn test_roundtrip_eth_transfer() -> Result<()> {
     assert_eq!(dai_supply, Some(transfer_amount));
 
     // let's transfer them back to Ethereum
-    let amount = token::DenominatedAmount {
-        amount: transfer_amount,
-        denom: 0u8.into(),
-    }
-    .to_string();
+    let amount =
+        token::DenominatedAmount::new(transfer_amount, 0u8.into()).to_string();
```

Comment:
I wouldn't bother touching the eth bridge e2e tests, honestly. they're still disabled